test/main/updater/manualCheck.test.js

```diff
@@ -0,0 +1,89 @@
+import log from 'electron-log'
+import nock from 'nock'
+
+import checkForUpdates from '../../../main/updater/manualCheck'
+import packageInfo from '../../../package.json'
+
+// response for current release
+const githubReleasesResponse = [{
+  html_url: 'https://frame.sh/the-next-great-release',
+  prerelease: false,
+  tag_name: packageInfo.version
+}]
+
+const currentVersion = packageInfo.version
+const nextVersion = currentVersion.slice(0, currentVersion.length - 1) + (parseInt(currentVersion[currentVersion.length -1]) + 1)
+
+beforeAll(() => {
+  nock.disableNetConnect()
+  log.transports.console.level = false
+})
+
+afterAll(() => {
+  nock.cleanAll()
+  nock.enableNetConnect()
+  log.transports.console.level = 'debug'
+})
+
+it('identifies that a newer version is not available', async () => {
+  mockApiResponse(200, githubReleasesResponse)
+
+  return expect(checkForUpdates()).resolves.toBeFalsy()
+})
+
+it('identifies that a newer version is available', async () => {
+  const response = [
+    {
+      html_url: 'https://frame.sh/cutting-edge-frame-release',
+      prerelease: true,
```

Comment:
do you need to do any tests for a new version which has prerelease set to false?

Reply:
thought about it but I think these tests cover all the cases since we can only be more restrictive about prereleases, ie if its not a prerelease it will always be picked up